webscraping/webscrape.py

```diff
@@ -0,0 +1,47 @@
+import http.client
+
+def firstBetween (string, startingString, endingString): #This function return whatever is between the starting string and ending string based off of an original string
+    idx = [i for i in range(len(string)) if string.startswith(startingString, i)]
+    if len(idx) > 0:
+        ret = exerciseHTML[idx[0]+ len(startingString):].split(endingString)[0]
+        return ret
+    return False
+
+def requestHTML(reqName):
+    conn = http.client.HTTPSConnection("www.acefitness.org")
+    conn.request("GET", reqName)
+    r1 = conn.getresponse()
+    print(r1.status, r1.reason)
+    data1 = r1.read()  # This will return entire content.
+    # The following example demonstrates reading data in chunks.
+    conn.request("GET",  reqName)
+    r1 = conn.getresponse()
+    OUTPUT = ""
+    with open('file.txt', 'w') as file:
```

Comment:
I wonder do we need to write it to file? Can we read it directly?

Reply:
Perhaps not necessary to write to a file, though I found it easier to test and debug when I did as I could have the scraped HTML in front of me while I coded! I could find a way to do this without the use of a file most likely.